src/components/Tooltip/index.tsx

```diff
@@ -3,10 +3,11 @@ import styled from 'styled-components'
 import { Button } from 'ariakit/button'
 import { Tooltip as AriaTooltip, TooltipAnchor, useTooltipState } from 'ariakit/tooltip'
 
-interface ITooltip {
+interface ITooltip<As = any> {
 	content: string | null
 	style?: {}
 	children: React.ReactNode
+	as?: As
```

Comment:
having a button in the table breaks the layout in the `RecentProtocols`, as default `Trigger` is `Button`
we need `as`, because we don't want to always have button triggers

(I hope it's fine that I use non-single-letter generics, but I think it's just cleaner)